src/flake8_simplify/plugins/use_contextlib_suppress.rs

```diff
@@ -0,0 +1,38 @@
+use rustpython_ast::{Excepthandler, ExcepthandlerKind};
+
+use crate::ast::helpers;
+use crate::checkers::ast::Checker;
+use crate::registry::{Check, CheckKind};
+use crate::source_code_locator::SourceCodeLocator;
+
+/// SIM105
+pub fn use_contextlib_suppress(
+    checker: &mut Checker,
+    handlers: &[Excepthandler],
+    locator: &SourceCodeLocator,
+) {
+    if handlers.len() != 1 {
+        return;
+    }
+    let handler = &handlers[0];
+    let ExcepthandlerKind::ExceptHandler { body, .. } = &handler.node;
+    if body.len() == 1 {
+        let node = &body[0].node;
+        if matches!(node, rustpython_ast::StmtKind::Pass) {
+            let handler_names: Vec<_> = helpers::extract_handler_names(handlers)
+                .into_iter()
+                .flatten()
+                .collect();
+            let exception = if handler_names.is_empty() {
+                "Exception".to_string()
+            } else {
+                handler_names.join(", ")
+            };
+            let check = Check::new(
+                CheckKind::UseContextlibSuppress(exception),
```

Comment:
This might be a hard one to autofix right now because it depends on `contextlib` being imported already.